config/elasticsearch/client.php

```diff
@@ -2,6 +2,7 @@
 
 return [
     'hosts' => [
-        sprintf('http://%s:%d', env('ES_HOST'), env('ES_PORT'))
+        'host' => env('ES_HOST', 'localhost'),
+        'port' => env('ES_PORT', '9200'),
```

Comment:
```suggestion
        [
                'host' => env('ES_HOST', 'localhost'),
                'port' => env('ES_PORT', '9200'),
        ]
```
I mean like this. Not sure if that would not be interpreted as two hosts: `localhost`, `9200`.

Reply:
You're totally right. This PR is already merged in, but here's the fix: https://github.com/SlovakNationalGallery/webumenia.sk/pull/380